scalafmt-core/shared/src/main/scala/org/scalafmt/util/LiteralOps.scala

```diff
@@ -0,0 +1,70 @@
+package org.scalafmt.util
```

Comment:
format writer is internal. format ops is there, too. should this be?

Reply:
But `TokenOps` is `util` :man_shrugging: 

In my mind `LiteralOps` is functions for printing literals independent of formatting context

Reply:
np, it was just a question